index.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-restricted-globals */
+
+const form = document.querySelector('#form');
+const Author = document.querySelector('.Author');
+const Title = document.querySelector('.Title');
+const bookText = document.getElementById('book-text');
+const container = document.querySelector('.container');
+const contact = document.querySelector('#contact');
+const displayContact = document.querySelector('#display_contact');
+const displayBooks = document.querySelector('#display_list');
+const displayForm = document.querySelector('#display_form');
+const date = document.querySelector('.date');
+
+class Book {
+  static validateInputData = (title, author) => {
+    if (title.length < 1 || author.length < 1) {
+      return false;
+    }
+    return true;
+  }
+
+  save = (title, author) => {
+    let books = [];
+    if (localStorage.getItem('book') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('book'));
+    }
+    const book = { text1: title, text2: author };
+    books.push(book);
+    localStorage.setItem('book', JSON.stringify(books));
+  }
+
+  static addBook = (title, author) => {
+    const isValidInput = this.validateInputData(title, author);
+    if (!isValidInput);
+    else {
+      const book = new Book(title, author);
+      book.save(title, author);
+      this.renderBooks();
+    }
+  }
+
+  static removeBook = (title, author) => {
+    let books = [];
+
+    if (localStorage.getItem('book') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('book'));
+    }
+    books.forEach((book, index) => {
+      if (book.text1 === title && book.text2 === author) {
+        books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('book', JSON.stringify(books));
+    this.renderBooks();
+  }
+
+  static renderBooks = () => {
+    bookText.innerHTML = '';
+    let books = [];
+    if (localStorage.getItem('book') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('book'));
+
+      books.forEach((book) => {
+        bookText.innerHTML += `
+            <div class="wrapper">
+              <p class="book-title">${book.text1}</p>
+              <p class="book-author">${book.text2}</p>
+              <button onclick="deleteBook('${book.text1}', 
+              '${book.text2}')">Remove</button>
```

Comment:
- [ ] An error 🐛 occurs when trying to delete a book. Kindly see the error attached below and try to fix it 👇 
![image](https://user-images.githubusercontent.com/46839548/206033713-f5e9720d-270f-4f6c-97e1-14c85544d233.png)

Reply:
I have resolved this. Thanks for the feedback